### PR TITLE
Update translation.id.json

### PR DESCRIPTION
**Pull request by Bangden**
Changing the language vocabulary with more courtesy.

### CODE REVIEW

**Comment by awaaas** [comment]
Nice. But please make `anda` in the middle of the sentence be lowercase.

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json [hunk 1]
```diff
@@ -270,7 +270,7 @@
     },
     {
       "Key": "gotUpToDateVersion",
-      "Value": "Sempurna! Kamu punya versi terbaru {0}"
+      "Value": "Info! Silahkan perbarui ke versi terbaru {0}"
```
Comment (awaaas) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: No. This is an information for the user that he got the latest version, please refer [this line.](https://github.com/NECROBOTIO/NecroBot/blob/master/PoGo.NecroBot.Logic/Common/Translations.cs#L325)

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json [hunk 2]
```diff
@@ -334,19 +334,19 @@
     },
     {
       "Key": "maxItemsCombinedOverMaxItemStorage",
-      "Value": "[Konfigurasi Salah] Total barang maksimum kamu (ball+potion+revive={0}) melebihi kapasitas tas kamu ({1})"
+      "Value": "[Konfigurasi Salah] Total barang maksimum Anda (ball+potion+revive={0}) melebihi kapasitas tas Anda ({1})"
     },
     {
       "Key": "recyclingQuietly",
-	  "Value": "Mendaur ulang senyap..."
+	  "Value": "Daur ulang dengan santai..."
```
Comment (awaaas) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: [The original line](https://github.com/NECROBOTIO/NecroBot/blob/master/PoGo.NecroBot.Logic/Common/Translations.cs#L356) is `quietly`, so `senyap` is right.

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json (hunk 1 above)
Comment (Bangden) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: Hello, nice to meet you. ^^
In that line, I tried to make sentences emphasis to the user to always use the latest version of the bot.

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json (hunk 2 above)
Comment (Bangden) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: Hello, @awaaas 
In this line, I saw the words were a little stiff. I intend to change the wording so as not to stiffen. Sorry if I am wrong. ^^

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json (hunk 1 above)
Comment (awaaas) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: Um... no, you're saying to the user that (s)he must update to newer version.

File: PoGo.NecroBot.CLI/Config/Translations/translation.id.json (hunk 2 above)
Comment (awaaas) [inline PoGo.NecroBot.CLI/Config/Translations/translation.id.json]: Yeah, but it makes a misleading translation though...

`Daur ulang dengan santai` -> `Relax recycling`, what does that even mean?

**Comment by brobotbott** [comment]
[help] issue loading translations
when I first language setup failed to change code id
what wrong ? sorry my speak english is bad :D
thanks before

**Pull request merged**